Task1

kojunsei opened the pull request:
6/12 kintone dojoの課題です。
レビューよろしくお願いいたします。

kojunsei committed:
123 txt file

kojunsei committed:
Delete task1 directory

kojunsei committed:
Delete 123.txt

kojunsei committed:
task1

H-Shira commented on task1/task1.html:
```diff
@@ -0,0 +1,42 @@
+    <main class>
+          <div class="kintone" align="center">
+            <br>
+            <img src="image-kn.png" alt="kintone" width="1024">
```
ブラウザで開いてみたところ、画像だけ表示ませんでした（ ;  ; ）

`<img>`要素の`src`属性を使って画像の位置を指定してもらってるんですが、こちらは参照するコンテンツを絶対パスor相対パスで指定してもらう必要があります。
今回、画像は「img」フォルダにまとめて保存していて、htmlを保存してる位置と異なっていると思うので、相対パスで画像の位置を指定してみてください！

H-Shira replied:
今回、全ての画像にそれぞれwidthを設定していると思います。
この画像たちは画像サイズもマージンも同じだと思うので、cssで一括で編集してみてください。
**全ての画像に同じclass名をつけてくれれば**、cssで一括で編集可能出来るかと！

H-Shira replied:
widthは単位なしで指定すると、「ピクセル」として高さを設定します（heightも同様です）。
これだと高さが固定になってしまい、ブラウザの横幅に合わせて可変にならないので、別の方法で設定してみてください。
▼これとか分かりやすいかもしれないです…！
https://jajaaan.co.jp/web-production/frontend/width-height/

H-Shira commented on task1/task1.html:
```diff
@@ -0,0 +1,42 @@
+          </div>
+          <div class="office" align="center">
+            <br>
+            <img src="image-of.png" alt="office" width="1024">
```
https://github.com/kojunsei/kintone_dojo/pull/1/commits/2024a0b0dd1be01f34d03aa86d301dfdd1434181#r1229169926
https://github.com/kojunsei/kintone_dojo/pull/1/commits/2024a0b0dd1be01f34d03aa86d301dfdd1434181#r1229177358
https://github.com/kojunsei/kintone_dojo/pull/1/commits/2024a0b0dd1be01f34d03aa86d301dfdd1434181#r1229256899
と同じです！

H-Shira commented on task1/task1.html:
```diff
@@ -0,0 +1,42 @@
+          </div>
+          <div class="garoon" align="center">
+            <br>
+            <img src="image-gr.png" alt="garoon" width="1024">
```
https://github.com/kojunsei/kintone_dojo/pull/1/commits/2024a0b0dd1be01f34d03aa86d301dfdd1434181#r1229169926
https://github.com/kojunsei/kintone_dojo/pull/1/commits/2024a0b0dd1be01f34d03aa86d301dfdd1434181#r1229177358
https://github.com/kojunsei/kintone_dojo/pull/1/commits/2024a0b0dd1be01f34d03aa86d301dfdd1434181#r1229256899
と同じです！

H-Shira commented on task1/task1.html:
```diff
@@ -0,0 +1,42 @@
+          </div>
+          <div class="mailwise" align="center">
+            <br>
+            <img src="image-mw.png" alt="mailwise" width="1024">
```
https://github.com/kojunsei/kintone_dojo/pull/1/commits/2024a0b0dd1be01f34d03aa86d301dfdd1434181#r1229169926
https://github.com/kojunsei/kintone_dojo/pull/1/commits/2024a0b0dd1be01f34d03aa86d301dfdd1434181#r1229177358
https://github.com/kojunsei/kintone_dojo/pull/1/commits/2024a0b0dd1be01f34d03aa86d301dfdd1434181#r1229256899
と同じです！

H-Shira commented on task1/task1.html:
```diff
@@ -0,0 +1,42 @@
+    </header>
+    <main class>
+          <div class="kintone" align="center">
+            <br>
```
https://github.com/kojunsei/kintone_dojo/pull/1/commits/2024a0b0dd1be01f34d03aa86d301dfdd1434181#r1229177358
と同じですが、htmlで`<br>` をつけるのでなく、cssで画像のクラスに`margin`を設定してみてください。

H-Shira commented on task1/task1.html:
```diff
@@ -0,0 +1,42 @@
+          <div class="kintone" align="center">
+            <br>
+            <img src="image-kn.png" alt="kintone" width="1024">
+            <br>
```
https://github.com/kojunsei/kintone_dojo/pull/1/commits/2024a0b0dd1be01f34d03aa86d301dfdd1434181#r1229177358
と同じですが、htmlで`<br>` をつけるのでなく、cssで画像のクラスに`margin`を設定してみてください（このhtmlのすべての`<br>`要素も同様です）。

H-Shira reviewed (changes requested):
お疲れ様です！レビューお待たせしました。
コメントに修正希望箇所を書きました。全てhtmlに記載してますが、一部cssの修正が必要なものもあります。
コメント箇所が多いので修正多いように感じられるかもしれないのですが、殆ど以下の三つです！
1. `<img>`の`src`のパスの書き方について
2. 画像をクラスで括って、css上で一括編集してみてください
3. width・heightの指定の仕方について(ブラウザで可変になるよう設定してみて)

わからないことあれば質問してください🙇‍♀️dojo頑張って〜！

kojunsei committed:
<img>のsrcでimgのファイルを指定した

ローカルで動くようになりました！
ご確認のほどよろしくお願いいたします

kojunsei committed:
width・heightの指定の仕方はpxではなく%に変更

リンクを拝見させていただきました！
vwという単位もございますが...あれがディスプレイを基準にしていますので、どっちかというと%のほうがブラウザーに合わせてくれると思い変更しました。

kojunsei committed:
Update task1.html finished

kojunsei committed:
Update task1 finished

kojunsei commented:
訂正が遅くなり申し訳ありません。
task.htmlとtask.cssを変更しました。
お手数をおかけしますが、
ご確認のほどよろしくお願いいたします。

☑<img>のsrcのパスの書き方について
☑画像をクラスで括って、css上で一括編集
☑width・heightの指定の仕方をpx→%に変更

H-Shira commented:
お疲れ様です！修正ありがとうございます
コードの内容は問題ないので、レビュー通過で大丈夫です！が、コンフリクトが起きているようです…💦
コンフリクトを解消してからマージしてみてください！

kojunsei committed:
Update task1.css
